### PR TITLE
Updated openshift version with the deployment changes in registry

### PR DESCRIPTION
**Pull request by bamachrn**
update openshift to 3.6 with router and registry enabled.
Fixed https://github.com/CentOS/container-pipeline-service/issues/318

### CODE REVIEW

**Comment by dharmit** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests-debug

**Comment by bamachrn** [comment]
ansible complains about rslave volume mounting. looking into it

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
openshift gets updated properly with all the bits running, but there is issue in template, it is saying as deprecated and not accepting. looking into that.

**Comment by bamachrn** [comment]
#dotests

File: container_pipeline/lib/openshift.py [hunk 1]
```diff
@@ -89,12 +89,12 @@ def clean_project(self, project):
                 project, e))
 
     def upload_template(self, project, template_path, template_data):
-        """Upload processed template for project from template path"""
+        """Upload processed template for project from template path."""
```
Comment (navidshaikh) [inline container_pipeline/lib/openshift.py]: why?

Comment (bamachrn) [inline container_pipeline/lib/openshift.py]: pytlint complains about putting dot in the first line of the comment

**Pull request merged**